### ISSUE TITLE
Privacy link doesn't work

### ISSUE THREAD

**Issue by gnunicorn** [acterglobal/a3#736]
Privacy Policy link doesnt work in register form.

**Comment by kumarpalsinh25** [comment]
@gnunicorn
Is this issue is still persists? Can you check and confirm?

**Comment by gnunicorn** [comment]
yeah, still not doing anything https://github.com/acterglobal/a3/blob/ben-event-rsvp-notifications/app/lib/features/auth/pages/register_page.dart#L369-L377
